Druid/DRInstruction.class.st

```diff
@@ -9,7 +9,8 @@ Class {
 		'origin',
 		'users',
 		'classType',
-		'isStaged'
+		'isStaged',
+		'scope'
```

Comment:
I prefer to have a subclass with the scope. Adding it to all instructions it's a bit overkill I think.